lib/api/unittest/CDataFrameAnalyzerFeatureImportanceTest.cc

```diff
@@ -536,6 +551,10 @@ BOOST_FIXTURE_TEST_CASE(testClassificationFeatureImportanceAllShap, SFixture) {
             c2Sum += std::fabs(c2);
             c3Sum += std::fabs(c3);
             c4Sum += std::fabs(c4);
+        } else if (result.HasMember("model_metadata")) {
+            if (result["model_metadata"].HasMember("total_feature_importance")) {
+                hasTotalFeatureImportance = true;
+            }
```

Comment:
Since this loop actually computes sum abs importances, I think it would be nice to check that the total feature importance values are correct as well.

Reply:
This could wait to the PR to reenable though.

Reply:
I extended the unit tests. However, since I temporarily commented out the output of the model metadata, the checks are also commented out. I also added a TODO to reactivate it, once the Java part is there.